core/src/test/java/io/snappydata/app/JavaCreateIndexTestSuite.java

```diff
@@ -0,0 +1,138 @@
+/*
```

Comment:
Do we want to put this into "app" package? That was originally meant for stand-alone many test programs and not junit tests. Perhaps something like io.snappydata.api will be better.